_data/simple-icons.json

```
@@ -7789,6 +7789,11 @@
             "hex": "D26B38",
             "source": "https://github.com/knex/documentation/blob/a9c4ce47dbc6001bb1c6aa0649bb668edc78fea7/src/public/knex-logo.png"
         },
+        {
+            "title": "KNIME",
+            "hex": "FDD800",
+            "source": "https://www.knime.com/about"
```

```suggestion
            "source": "https://www.knime.com"
```

SVG is available in header of homepage.